rsyslog/init.sls

```diff
@@ -11,21 +11,42 @@ stoplogger_{{logger}}:
 {% endfor %}
 {% endif %}
 
+remove_old_rsyslog:
+  pkg.purged:
+    - version: 8.16.0-1ubuntu3.1
+
 rsyslog:
+  pkgrepo.managed:
+    - name: deb http://download.opensuse.org/repositories/home:/rgerhards/xUbuntu_16.04/ /
+    - key_url: https://download.opensuse.org/repositories/home:rgerhards/xUbuntu_16.04/Release.key
   pkg.installed:
-    - name: {{ rsyslog.package }}
+    - name: rsyslog
+    - version: 8.2001.0-1
   file.managed:
     - name: {{ rsyslog.config }}
     - template: jinja
     - source: {{ rsyslog.custom_config_template }}
     - context:
         config: {{ rsyslog|json }}
+
+rsyslog_omhttp
```

Comment:
Just curious with this package - it looks like there are a bunch of config params: https://www.rsyslog.com/doc/master/configuration/modules/omhttp.html

But I don't see any in this PR. How does `om-http` know which server and port to send the logs to, for example? Is this a config that is carried over from fluentd or something? Or would this be in the `salt-states` repo instead since that's where we do configuration and this is just a formula? Sorry for my naiitivity - just starting to figure out how all this salt stuff is fitting together.